ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +76,177 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status_ok(replaying_images=None):
+    """
+    Check if mirroring status has health OK and expected number of replaying images
```

Comment:
@sidhant-agrawal Could you please add a line about context of the cluster in which we are running! For example , looking at this function I am not sure if it runs on ACM OR Primary and secondary. Please do the same wherever applicable.

Reply:
This function doesn't change the context.This will run in whichever context the test is running.
Another function `wait_for_mirroring_status_ok` will use this function to check mirroring status on both ODF cluster and that will take care of changing context.

Reply:
ack